backend/TikTakToe.Engines/Engines/random/random.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TikTakToe.Core.Boards;
+
+namespace TikTakToe.Engines.Engines.random
+{
+    public class random : IEngine
+    {
+        public Core.Enums.Engines Engine => Core.Enums.Engines.Random;
+
+        private int board;
+        private Stopwatch sw = new Stopwatch();
+        private Random randomPicker = new Random();
+
+        public double Bench(int depth)
+        {
+            sw.Restart();
+            sw.Start();
```

Comment:
restart and start at the same time?

Reply:
it work, but has been changed anyway as discussed before